### DISCUSSION TITLE
[Suggestion] Allow YouTube embeds in READMEs

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#48753] [Repositories]
### Select Topic Area

Product Feedback

### Body

This would be amazing & useful, and most other sites allow it

**Comment by YasinDehfuli** [comment]
First, find the YouTube video you want to embed and click the Share button below the video. Then, click on the Embed button and copy the HTML code that appears.

Next, open your README.md file on GitHub and paste the HTML code where you want the YouTube video to appear. Make sure to switch to the HTML editor mode in the GitHub text editor before pasting the code.

Once you've pasted the code, save the changes to the README.md file and commit the changes to the repository.

When you or someone else views the README.md file on GitHub, the YouTube video should be embedded and playable directly in the document.

It's important to note that embedding YouTube videos in READMEs can impact the load time of your page, so be mindful of how many videos you embed and their size.

I hope this way helps you !

**Comment by HashimTheArab** [comment]
You can do this using https://githubvideo.com

**Comment by Jeiel0rbit** [comment]
> You can do this using https://githubvideo.com

Alternative? It seems to me that it is loading slowly.

**Reply by h3rb** [reply]
Site is gone now

**Reply by simonprovost** [reply]
Site still gone

**Comment by Jeiel0rbit** [comment]
> First, find the YouTube video you want to embed and click the Share button below the video. Then, click on the Embed button and copy the HTML code that appears.
> 
> Next, open your README.md file on GitHub and paste the HTML code where you want the YouTube video to appear. Make sure to switch to the HTML editor mode in the GitHub text editor before pasting the code.
> 

On mobile phones there could be an option to detect YouTube URLs and create the HTML code.

**Comment by sionnx** [comment]
you should try this: https://t.cuts.so/github/video

**Reply by Gab91s** [reply]
thank you

**Comment by tade56777** [comment]
Use your YOUTUBE LINK and but it in the README

**Comment by LeonardoTemperanza** [comment]
Contrary to what people are saying here, this is just not possible. Videos can be embedded if uploaded to github, but you have a 10MB limit there which is peanuts. Seems like a basic feature to me, weird that it's still missing.

**Comment by tips-tricks-be** [comment]
[![YouTube](http://i.ytimg.com/vi/dWUyvIB3DgM/hqdefault.jpg)](https://www.youtube.com/watch?v=dWUyvIB3DgM)

[![YouTube](http://i.ytimg.com/vi/PUT HERE CODE/hqdefault.jpg)](https://www.youtube.com/watch?v=PUT HERE CODE)
replace the code whit the code from your youtube link
